system/classes/Kohana/View.php

```diff
@@ -233,19 +228,19 @@ public function __toString()
 	{
 		try
 		{
-			return $this->render();
+			return (string) $this->render();
```

Comment:
The render() method returns a string.

Reply:
render method at errors can be return void/null

Reply:
https://github.com/koseven/koseven/blob/ec1778f1fe9175fdb07f1babfedcd37716cc4c67/system/classes/Kohana/View.php#L359
https://github.com/koseven/koseven/blob/ec1778f1fe9175fdb07f1babfedcd37716cc4c67/system/classes/Kohana/View.php#L74

According to the documentation
```
/**
 * Get current buffer contents and delete current output buffer
 * @link http://php.net/manual/en/function.ob-get-clean.php
 * @return string the contents of the output buffer and end output buffering.
 * If output buffering isn't active then false is returned.
 * @since 4.3.0
 * @since 5.0
 */
function ob_get_clean () {}
```

Reply:
@bkosun search problems then method return not string harder than fix returned value.